frontend/src/helper.js

```diff
@@ -2,6 +2,8 @@ import i18n, { getBrowserLang } from './i18n.js'
 
 const configEnv = require('../configEnv.json')
 
+export const TRACIM_APP_VERSION = 'v2.2'
```

Comment:
I would prefer (if possible) to have a version.js file including online this part. This way, it's clear for the release manager that version must be updated in version.js file

Reply:
ok